.goreleaser.yml

```diff
@@ -110,3 +110,13 @@ dockers:
     - "sonatypecommunity/nancy:{{ .Tag }}-alpine"
     - "sonatypecommunity/nancy:v{{ .Major }}-alpine"
     - "sonatypecommunity/nancy:v{{ .Major }}.{{ .Minor }}-alpine"
+
+nfpms:
+  -
+    vendor: sonatype-nexus-community
+    homepage: https://github.com/sonatype-nexus-community/nancy
+    description: "A tool to check for vulnerabilities in your Golang dependencies, powered by Sonatype OSS Index"
+    formats:
+      - apk
```

Comment:
So this will just build them and attach them to the release, yeah? Do we have a plan for distributing them further? Just curious!

Reply:
I think we should that would be neat and same goes for ahab :)

Reply:
I just opened #177 to address that.

Reply:
and this one for ahab :) https://github.com/sonatype-nexus-community/ahab/issues/38